decidim-admin/app/controllers/decidim/admin/concerns/participatory_process_admin.rb

```diff
@@ -0,0 +1,31 @@
+# frozen_string_literal: true
+module Decidim
+  module Admin
+    module Concerns
+      # This concern is meant to be included in all controllers that are scoped
+      # into a participatory process' admin panel. It will override the layout
+      # so it shows the sidebar, preload the participatory process, etc.
+      module ParticipatoryProcessAdmin
+        extend ActiveSupport::Concern
+
+        included do
+          helper_method :participatory_process
+          before_filter :ensure_participatory_process
+
+          layout "decidim/admin/participatory_process"
+        end
+
+        private
+
+        def participatory_process
+          @participatory_process ||=
+            current_organization.participatory_processes.find(params[:participatory_process_id])
+        end
+
+        def ensure_participatory_process
+          false unless participatory_process
```

Comment:
Return false won't have any effect, in other parts of the code we've done this:

`raise ActionController::RoutingError, "Not Found" unless participatory_process`